drivers/console/bm_serial.c

```diff
@@ -540,6 +401,7 @@ int bm_serial_frm_put(bm_frame_t* bm_frm)
 {
 	int retval = -1;
 	uint16_t frame_length = bm_frm->frm_hdr.payload_length + sizeof(bm_frame_header_t);
```

Comment:
Same here, should we check that frame_length+crc size <= size of the `tx_payload_buf`